modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -81,26 +81,6 @@ public void testBuildTemplateInWorkspaceWithPackageName() throws Exception {
 			"--dependency-management-enabled");
 	}
 
-	@Test(expected = IllegalArgumentException.class)
-	public void testBuildTemplateLiferayVersionInvalid62() throws Exception {
-		_buildTemplateWithGradle(
-			"mvc-portlet", "test", "--liferay-version", "6.2");
-	}
-
-	@Test(expected = IllegalArgumentException.class)
-	public void testBuildTemplateLiferayVersionInvalid70test()
-		throws Exception {
-
-		_buildTemplateWithGradle(
-			"mvc-portlet", "test", "--liferay-version", "7.0test");
-	}
-
-	@Test(expected = IllegalArgumentException.class)
-	public void testBuildTemplateLiferayVersionInvalidTest() throws Exception {
-		_buildTemplateWithGradle(
-			"mvc-portlet", "test", "--liferay-version", "test.test");
```

Comment:
Does this mean that a version of test.test will no longer throw an exception? It should still right?